tests/sentry/tasks/test_post_process.py

```diff
@@ -1106,7 +1107,8 @@ def test_debounce_cache_is_set(self, mock_get_commit_context):
         return_value=github_blame_return_value,
     )
     def test_logic_fallback_no_scm(self, mock_get_commit_context):
-        Integration.objects.all().delete()
+        with in_test_psql_role_override("postgres"):
```

Comment:
It's cool that the deletion test user catches this, which gives me greater confidence we're not missing any deletes.